src/modules/webpack-publish.test.ts

```diff
@@ -0,0 +1,113 @@
+import { Factory } from "rosie";
+import { WebpackRestoreOptions } from "../interfaces/webpack-restore-options";
+import { FactoryType } from "../tests/factories/factory-type";
+import { NodeCI } from "./node-ci";
+import { NodeClean } from "./node-clean";
+import { NodeRestore } from "./node-restore";
+import { WebpackPublish } from "./webpack-publish";
+
+describe("webpack-publish", () => {
```

Comment:
Minor: I think we've been naming the top-level describes for modules as their exported values, ie `WebpackPublish`